crates/bevy_render/src/render_graph/graph.rs

```diff
@@ -676,6 +714,22 @@ mod tests {
         }
     }
 
+    fn input_nodes(name: &'static str, graph: &RenderGraph) -> HashSet<NodeId> {
```

Comment:
This code was simply moved out of test_graph_edges because I used it in the new test.